[2024-12-27T04:56:55.603+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-27T04:56:55.718+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_kafka_pipeline.fetch_and_produce_data scheduled__2024-12-27T03:00:00+00:00 [queued]>
[2024-12-27T04:56:55.828+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_kafka_pipeline.fetch_and_produce_data scheduled__2024-12-27T03:00:00+00:00 [queued]>
[2024-12-27T04:56:55.876+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 3
[2024-12-27T04:56:56.505+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_and_produce_data> on 2024-12-27 03:00:00+00:00
[2024-12-27T04:56:56.714+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=15418) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-27T04:56:56.796+0000] {standard_task_runner.py:72} INFO - Started process 15440 to run task
[2024-12-27T04:56:56.711+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'api_to_kafka_pipeline', 'fetch_and_produce_data', 'scheduled__2024-12-27T03:00:00+00:00', '--job-id', '110', '--raw', '--subdir', 'DAGS_FOLDER/api_to_kafka_dag.py', '--cfg-path', '/tmp/tmpsotv5fr0']
[2024-12-27T04:56:56.809+0000] {standard_task_runner.py:105} INFO - Job 110: Subtask fetch_and_produce_data
[2024-12-27T04:56:57.039+0000] {task_command.py:467} INFO - Running <TaskInstance: api_to_kafka_pipeline.fetch_and_produce_data scheduled__2024-12-27T03:00:00+00:00 [running]> on host bd7ba946b34c
[2024-12-27T04:56:58.411+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='api_to_kafka_pipeline' AIRFLOW_CTX_TASK_ID='fetch_and_produce_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-27T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-27T03:00:00+00:00'
[2024-12-27T04:56:58.497+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-27T04:56:59.897+0000] {api_to_kafka_dag.py:85} INFO - Fetching data for IBM at 5min interval
[2024-12-27T05:23:22.372+0000] {api_to_kafka_dag.py:95} INFO - Processing and sending data to Kafka
[2024-12-27T05:23:26.361+0000] {job.py:229} INFO - Heartbeat recovered after 1591.25 seconds
[2024-12-27T05:47:44.832+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to None. Terminating instance.
[2024-12-27T05:47:44.949+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-12-27T05:47:44.958+0000] {process_utils.py:132} INFO - Sending 15 to group 15440. PIDs of all processes in the group: [15440]
[2024-12-27T05:47:44.963+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 15440
[2024-12-27T05:48:44.987+0000] {process_utils.py:150} WARNING - process psutil.Process(pid=15440, name='airflow task ru', status='sleeping', started='04:56:55') did not respond to SIGTERM. Trying SIGKILL
[2024-12-27T05:48:44.995+0000] {process_utils.py:87} INFO - Sending the signal 9 to group 15440
[2024-12-27T05:48:45.010+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=15440, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='04:56:55') (15440) terminated with exit code -9
[2024-12-27T05:48:45.013+0000] {standard_task_runner.py:190} ERROR - ('Job 110 was killed before it finished (likely due to running out of memory)', 'For more information, see https://airflow.apache.org/docs/apache-airflow/stable/troubleshooting.html#LocalTaskJob-killed')
